tests/test_feature_extraction.py

```
@@ -141,4 +141,36 @@ def test_4DLung_radiomicFeatureExtraction_output(lung4DMetadataPath):
                                        roiNames = "Tumor_c40",
                                        outputDirPath = "tests/4D-Lung/results/")
     expected_path = "tests/4D-Lung/results/features/radiomicfeatures_original_4D-Lung.csv"
-    assert os.path.exists(expected_path)
\ No newline at end of file
+    assert os.path.exists(expected_path)
+
+
+def test_segmentationLabel_passed(nsclcCTImage, nsclcSEGImage):
+    """Test passing in segmentation label"""
+    actual = singleRadiomicFeatureExtraction(nsclcCTImage, nsclcSEGImage, segmentationLabel=255)
+    assert type(actual) == collections.OrderedDict, \
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)